extensions/amp-story-auto-ads/0.1/story-ad-ui.js

```diff
@@ -182,6 +192,63 @@ export function handleAttributionClick(win, href) {
   openWindowDialog(win, href, '_blank');
 }
 
+/**  Creates a page-outlink element, returns an anchor tag containing relevant data if successful
```

Comment:
nit: description should start on second line of multiline comments https://jsdoc.app/about-getting-started.html